api.swagger.yaml

```diff
@@ -1327,6 +2254,233 @@ components:
         - email
         - username
       type: object
+    user-search-schema:
+      allOf:
+        - $ref: '#/components/schemas/user-schema'
+        - type: object
+          properties:
+            user_uuid:
+              description: This is the uuid of the user
+              type: string
+              example: 1f0c6cb5-a3c3-4197-bebc-b3e44462dc23
+    common-status-schema:
+      properties:
+        status:
+          description: A boolean flag
+          type: boolean
+          example: true
+    group-schema:
+      properties:
+        title:
+          description: This is the name of the group
+          type: string
+          example: Security awareness
+        description:
+          description: This is a description of the group.
+          type: string
+          example: A group to discuss security awareness.
+        group_category:
+          description: The category for the group.
+          type: string
+          example: Security
+        group_category_uuid:
+          description: The uuid of the category
+          type: string
+          pattern: >-
+            ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+          example: 01234567-89ab-cdef-1234-56789abcdef0
+        privacy_setting:
+          description: This is the privacy setting for the group.
+          type: string
+          enum:
+            - Public
+            - Private
+            - Secret
```

Comment:
I wonder if these should just be lowercase? As in this context they're effectively machine names

Reply:
agreed/updated